hazelcast/src/main/java/com/hazelcast/instance/EndpointQualifier.java

```diff
@@ -85,10 +85,6 @@ public boolean equals(Object o) {
             return false;
         }
 
-        // Single instance types - identifier doesn't matter
-        if (type.getServerSocketCardinality() == 1) {
```

Comment:
@leszko we were not checking the `identifier` for equality when getServerSocketCardinality is 1 before
because the only one with a different `identifier` was WAN. Now the CLIENT can have a different `identifier` and its cardinality is one. That is why we needed to remove this.

Reply:
I see. Then this change shouldn't break any WAN-related code.